XX-eltoo-transactions.md

```diff
@@ -125,11 +130,11 @@ where
 
 where EXPR_UPDATE(n) =
 
-`1 OP_CHECKSIGVERIFY <n> OP_CLTV`
+`and_v(v:pk(1),after(n))`
 
 and where EXPR_SETTLE(n) =
 
-`CovSig(n) 1_G OP_CHECKSIG`
+`CovSig(n) ||  pk(1_G)`
```

Comment:
yeah left it here in case it made sense... if we're just doing script here, why not just do script everywhere(with optimized miniscript-compatible bitcoin scripts)?

Reply:
The thought I had originally was to provide both the script and the miniscript (and/or miniscript policy?) used to generate the script...

This one's kind of a special case in any event, I think

Reply:
it's redundant information; you can encode it either way losslessly

Reply:
policy to miniscript isn't lossless. it's more that miniscript is more readily/obviously type-checkable? *shrug*

miniscript > script > making up your own syntax, I guess.

Reply:
either meaning: miniscript or bitcoin script, not policy

Reply:
maybe including both the policy definition (informational) and the script it compiled to (the actual spec/standard) would work?